lib/main.js

```diff
@@ -288,14 +292,21 @@ if (!process.env.CI) {
     })
   })
 
-  ava.test('supervisor: reload supervisor on a running device', async (test) => {
+  ava.test('supervisor: Reload supervisor on a running device', async (test) => {
     return utils.runManualTestCase(test, {
-      prepare: [ 'Ensure the device is running an application' ],
+      description: [
+        'Supervisor: Reload supervisor on a running device',
+        'testing if the supervisor can be deployed on a device that has lost its own' ],
+      prepare: [
+        'Ensure the device is online and running an application',
+        'Logging into the device (ssh to the host OS)',
+        'Check with "balena images" that the correct supervisor version is running on the device'
+      ],
       do: [
-        'Restart application from the dashboard',
```

Comment:
@horia-delicoti Can you create a GH issue about restarting the application by default before each test? (ie on a `beforeEach` clause?)

Reply:
done